### PR TITLE
Mobile version-Wotk section-about-section

### PR DESCRIPTION
**Pull request by kinginthenorthcodez**
#Middle Section
-Added Work section
-Fixed linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -79,5 +79,105 @@ <h3>Skills</h3>
...
+             has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.
+         </p>
+         </div>
+          <ul class="categories">
+             <li>css</li>
+             <li>html</li>
+             <li>Bootstrap</li>
+             <li>Ruby</li>
+          </ul>
```
Comment (aimemalaika) [inline index.html:106]: - [ ] Kindly add a separator between technologies as shown in the Figma design.
Figma
Screen Shot 2022-03-18 at 16 33 19
Your design
Screen Shot 2022-03-18 at 16 33 43

File: assets/css/style.css [hunk 2]
```diff
@@ -213,3 +228,164 @@ body {
...
+  width: 327px;
+}
+
+.card #firstTitle {
+  background: url(../images/EllipseFT.svg);
+  background-size: 150px 482px;
+  background-repeat: no-repeat;
+  background-position-y: 100%;
+  background-position-x: 0%;
+}
```
Comment (aimemalaika) [inline assets/css/style.css]: - [ ] Kindly make sure that the background touches the edge of the page as shown in the design as shown in the Figma design
- Figma
Screen Shot 2022-03-18 at 16 38 22
- Your design
Screen Shot 2022-03-18 at 16 34 35

**Review by aimemalaika — CHANGES_REQUESTED** [review]
Hi @kinginthenorthcodez, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
- Nice PR description ✅ 
- Nice use of flex and grid ✅ 

## Required Changes ♻️

- [ ] Check the comment below for required modifications

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (kinginthenorthcodez) [reply index.html:106]: Added separator  👍🏽

File: assets/css/style.css (hunk 2 above)
Comment (kinginthenorthcodez) [reply assets/css/style.css]: Fixed background

**Review by nicupop729 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hi @kinginthenorthcodez, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


well-1

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kinginthenorthcodez** [comment]
Thanks for that  review 👍🏽

**Pull request merged**